src/Razor/test/Microsoft.VisualStudio.Editor.Razor.Test/DefaultRazorIndentationFactsServiceTest.cs

```diff
@@ -194,9 +194,9 @@ public void GetDesiredIndentation_ReturnsNull_IfOwningSpanDoesNotExist()
     public void GetDesiredIndentation_ReturnsNull_IfOwningSpanIsCode()
     {
         // Arrange
-        var source = new StringTextSnapshot($@"
-@{{
-");
+        var source = new StringTextSnapshot("""
+@{
+""");
```

Comment:
Nit
```suggestion
            @{
            """);
```

Reply:
Looks like a few more cases of this, and the above issue, in these files. Maybe some bugs in your tool 😛

Reply:
I intentionally didn't adjust the indentation here. I can if you really want me to.